### ISSUE TITLE
Probe Basic Lathe: documentation 

### ISSUE THREAD

**Issue by Gnevko** [kcjengr/probe_basic#27]
All type of documentation are missing:
1) How to install
2) How to configure
3) How to use

For people who coms from "Mill World" it's not really easy to understand, how a Cnc lathe should be configured and how to use for example tool touch and so on.

**Comment by Lcvette** [comment]
this is true!!  you seem very active on this, would you mind maybe assisting in coming up with some documentation?

**Comment by Lcvette** [comment]
> All type of documentation are missing:
> 
> 1. How to install
> 2. How to configure
> 3. How to use
> 
> For people who coms from "Mill World" it's not really easy to understand, how a Cnc lathe should be configured and how to use for example tool touch and so on.

this is true!! you seem very active on this, would you mind maybe assisting in coming up with some documentation?
